search/include/pcl/search/flann_search.h

```diff
@@ -363,6 +363,8 @@ namespace pcl
         Indices index_mapping_;
         bool identity_mapping_;
 
+        int total_nr_points_;
```

Comment:
Please use `std::size_t` or `uindex_t` (like in `KdTreeFLANN`; `uindex_t` resolves to `unsigned int` by default but can be configured to be `unsigned long int` instead). We have had reports where people deal with so many points that an `int` is too small to hold the number of points.
Using an unsigned type here might make casting `k` necessary to avoid a signed-unsigned-comparison warning.